mautic/mautic
# Fix tags HTML-escaping

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/6057
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This fix is based on issues https://github.com/mautic/mautic/issues/6057 
This PR fix HTML-escaping for tags 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create a tag with the tracking script with a special character  and refresh page

    `mt('send', 'pageview', { tags: ['-test"eee', 'test"fff' ]});`

2. Too much  rows gets created in the database in lead_tags table


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Repeat all steps 
3. Should now work properly
